packages/preview/modern-unimi-thesis/0.1.0/README.md

```diff
@@ -0,0 +1,63 @@
+# modern-unimi-thesis 🎓
+
+This template is the result of the conversion from the [original LaTeX template](https://www.overleaf.com/project/641879675262cde2a670826b), in Italian.
```

Comment:
A bit more context about what the template is for would be appreciated (just giving the name of the University would be enough).